bigtable/client/table.h

```diff
@@ -18,14 +18,18 @@
 #include "bigtable/client/data_client.h"
 #include "bigtable/client/filters.h"
 #include "bigtable/client/idempotent_mutation_policy.h"
+#include "bigtable/client/internal/unary_rpc_utils.h"
 #include "bigtable/client/mutations.h"
+#include "bigtable/client/read_modify_write_rule.h"
 #include "bigtable/client/row_reader.h"
 #include "bigtable/client/row_set.h"
 #include "bigtable/client/rpc_backoff_policy.h"
 #include "bigtable/client/rpc_retry_policy.h"
 
 #include <google/bigtable/v2/bigtable.grpc.pb.h>
 
+namespace btproto = ::google::bigtable::v2;
```

Comment:
Please remove, no namespace aliases (or opening namespaces) in the header files.